specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachineScaleSet.json

```diff
@@ -3618,6 +3621,26 @@
       },
       "description": "Specifies the Spot-Try-Restore properties for the virtual machine scale set. <br><br> With this property customer can enable or disable automatic restore of the evicted Spot VMSS VM instances opportunistically based on capacity availability and pricing constraint."
     },
+    "PriorityMixPolicy": {
+      "type": "object",
+      "properties": {
+        "baseRegularPriorityCount": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "maximum": 100,
```

Comment:
The max isn't 100 -- this is a count (e.g., like `sku.capacity`), not a percentage like the other field.

Reply:
Is there no maximum for the baseRegularPriority? On page 9 of the SpotFleet Design Doc, it states that 100 is the maximum value allowed, even for the base regular priority count https://microsoft.sharepoint.com/:w:/t/ComputeVM/EdTHsdCkmMxMrGxZUWpl4pYBn3NP7dVRCDJg8SDWTIkouA?e=HC9dwu

Reply:
Great catch - that line is wrong :) (Looks like a copy/paste of the line for the percentage.)

The maximum on this value is the same as the maximum on `sku.capacity` (which is to say, in terms of the API schema definition that we can provide in this spec, there is no maximum).